src/v3/src/transformer/layout/idxTransformerMapping.ts

```diff
@@ -301,6 +302,22 @@ const TransformerMap: {
       buttonConfig: { showDefaultSubmit: false },
     },
   },
+  [IDX_STEP.DEVICE_APPLE_SSO_EXTENSION]: {
+    [AUTHENTICATOR_KEY.DEFAULT]: {
+      transform: transformAppleSsoExtension,
+      buttonConfig: { showDefaultSubmit: false },
+    },
+    [AUTHENTICATOR_KEY.OV]: {
+      transform: transformAppleSsoExtension,
+      buttonConfig: { showDefaultSubmit: false },
+    },
+  },
+  [IDX_STEP.CANCEL_TRANSACTION]: {
+    [AUTHENTICATOR_KEY.DEFAULT]: {
+      transform: transformAppleSsoExtension,
+      buttonConfig: { showDefaultSubmit: false },
+    },
+  },
```

Comment:
Would it be possible to order these keys alphabetically in this file like the others?